styles/components/_gallery-item.scss

```diff
@@ -54,10 +53,37 @@
     @include ktheme() {
       color: theme('k-blue') !important;
       &:hover {
-        color: theme('k-blue-hover')  !important;
+        color: theme('k-blue-hover') !important;
       }
     }
   }
 
   word-break: 'break-word';
 }
+
+.buy-tooltip {
+  width: max-content !important;
+
+  a {
+    cursor: pointer;
+    @include ktheme() {
+      color: theme('k-blue') !important;
+      &:hover {
+        color: theme('k-blue-hover') !important;
+      }
+    }
+  }
+}
+
+.gallery-item-disabled {
+  @include ktheme() {
+    box-shadow: 4px 4px theme('k-grey');
+    width: 10rem;
+  }
+
+  .neo-tooltip,
+  .is-neo {
+    height: 54px;
```

Comment:
is it necessary to have a fixed height?

Reply:
yeah else the button will be of a different height